lessons/pydata/pandas_groupby/index.ipynb

```diff
@@ -0,0 +1,3297 @@
+{
```

Comment:
<p>Asi bych začal jednoduší variantou <code>size</code> - dost často nám jsou nany ukradené, ale naopak chceme znát počet řádků v groupby.</p><p> </p><p>Ale pak samozřejmě nebude relevantní diskuse o chybějících hodnotách. Tak nevím...</p><p> </p><p>Taky můžeš <code>size</code>  uvést níže, u gender_counts, tam si to vyloženě říká.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Přidal jsem size za to, abychom tam měli obojí.</p>